### PR TITLE
[FEATURE] Send Progress Report via WhatsApp

### PR DESCRIPTION
**Pull request by anvmn**
#292

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_894

**Comment by anvmn** [comment]
Peek 2022-07-06 21-35

**Comment by anvmn** [comment]
Peek 2022-07-09 00-06

**Comment by anvmn** [comment]
Peek 2022-07-10 14-06

**Comment by anvmn** [comment]
Peek 2022-07-10 21-21

**Comment by anvmn** [comment]
@AronNovak please review.
Since there's another feature that we may need to deliver before this one, please **DO NOT MERGE** this PR,
as this is a base branch for the feature.
Also, there're couple of @todos here, that are handled in follow up PRs.

File: client/src/elm/Translate.elm
Comment (AronNovak) [inline client/src/elm/Translate.elm]:
```suggestion
            { english = "Is this the correct number for the patient's WhatsApp?"
```

File: client/src/elm/Translate.elm
Comment (AronNovak) [inline client/src/elm/Translate.elm:15253]: If it would be PHP + Drupal, we'd complain that this is not really translatable. Not sure about Kinyarwanda language, but splitting a sentence into chucks typically cause translatability issues.

Comment (anvmn) [reply client/src/elm/Translate.elm:15253]: @AronNovak The thing is that there requiremetn is to have patient name and phone number as part of the sentence, and they should have blue color:

Selection_1462

This means that we need to wrap the sentence with HTML:
```
div [ class "content" ]
        [ translateText language Translate.SendViaWhatsAppPhoneUpdateAtProfileQuestionPrefix
        , span [] [ text person.name ]
        , translateText language Translate.SendViaWhatsAppPhoneUpdateAtProfileQuestionSuffix
        , span [] [ text phoneNumber ]
        , text "?"
        ]
```

Since our Translate works with pure strings right now, we implement it this way.
When translation will be added, we'll review alternatives, if current structure is problematic.

**Review by AronNovak — APPROVED** [review]

**Pull request merged**
